google-postgresql.yml

```diff
@@ -39,13 +39,16 @@ provision:
     constraints:
       pattern: ^POSTGRES_[0-9]+$
   - field_name: storage_gb
```

Comment:
We could choose to set `prohibit_update` to true here. That would make it explicit that the property cannot be updated. 
However, if we did that the customers would also not be able to move from one plan to another if the plan defines the `storage_gb` as well (which I think is most of thee real use cases).